central/cve/image/service/service_impl.go

```diff
@@ -73,6 +77,9 @@ func (s *serviceImpl) SuppressCVEs(ctx context.Context, request *v1.SuppressCVER
 
 // UnsuppressCVEs un-suppresses given image CVEs.
 func (s *serviceImpl) UnsuppressCVEs(ctx context.Context, request *v1.UnsuppressCVERequest) (*v1.Empty, error) {
+	if features.UnifiedCVEDeferral.Enabled() {
+		return nil, errox.NotFound
+	}
```

Comment:
We'll need to add UI checks to hide the relevant components when this flag is enabled as well.

Will the legacy globally deferred CVEs be migrated to the new deferral type when this feature goes live? If not, indefinitely "snoozed" CVEs will no longer be accessible in the system.

Reply:
> Will the legacy globally deferred CVEs be migrated to the new deferral type when this feature goes live?


Yes.

 @dvail when do you estimate feature flag checks on the front end to be available on master?

Reply:
I don't have a current estimate. I did create a ticket for the UI team to track this work though https://issues.redhat.com/browse/ROX-20168

Reply:
Okay, please comment here once the frontend blockers are out of the way.

Reply:
[PR](https://github.com/stackrox/stackrox/pull/8216) to remove the global snooze UI for Image CVEs in VM 1.0 is up for review.